docs/configuration.md

```diff
@@ -188,8 +188,8 @@ By default, TorchServe uses all available GPUs for inference. Use `number_of_gpu
 Most of the following properties are designed for performance tuning. Adjusting these numbers will impact scalability and throughput.
 
 * `enable_envvars_config`: Enable configuring TorchServe through environment variables. When this option is set to "true", all the static configurations of TorchServe can come through environment variables as well. Default: false
-* `number_of_netty_threads`: number frontend netty thread. Default: number of logical processors available to the JVM.
-* `netty_client_threads`: number of backend netty thread. Default: number of logical processors available to the JVM.
+* `number_of_netty_threads`: number frontend netty thread. This specifies the numer of threads in the child [EventLoopGroup](https://livebook.manning.com/book/netty-in-action/chapter-8) of the frontend netty server. This group provides EventLoops for processing Netty Channel events (namely inference and management requests) from accepted connections. Default: number of logical processors available to the JVM.
```

Comment:
Maybe add a bit of background about how Netty is used in TorchServe

Reply:
Yes that will be helpful to get some clarity on the internal workings and how to think about tuning the different threads related configuration parameters